conf/etc/v2/cluster_health_hierarchy.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Should we be using the class instead? It must be available to users to know the hierarchy, probably the part of HealthHierarchy class.

Reply:
Sure, will modify in the refactor PR.